go/src/meguca/db/migrations.go

```diff
@@ -716,17 +716,76 @@ var migrations = []func(*sql.Tx) error{
 	},
 	// Fixes global moderation
 	func(tx *sql.Tx) (err error) {
-		err = WriteBoard(tx, BoardConfigs{
+		err = WriteBoardV(tx, BoardConfigs{
```

Comment:
Really, just inline it here.

Reply:
Pre-mature optimization is the root of all evil.
I just realized that was pointless since it calls `db.version` each time anyway.

Reply:
Less about premature optimization on more about not defining functions you use once in the global context.